charts/sn-platform/values.yaml

```diff
@@ -1243,16 +1245,19 @@ broker:
     extraOptions: []
     gcLoggingOptions: []
   configData:
-    # @deprecated:
-    # 1. You can use a service account to assume an IAM role to access S3 bucket
-    # 2. You can use a k8s secret to reference AWS credentials. See `offload.s3.secret`
-    # AWS_ACCESS_KEY_ID: "[YOUR AWS ACCESS KEY ID]"
-    # AWS_SECRET_ACCESS_KEY: "[YOUR SECRET]"
     managedLedgerDefaultEnsembleSize: "3"
     managedLedgerDefaultWriteQuorum: "3"
     managedLedgerDefaultAckQuorum: "2"
     PULSAR_PREFIX_systemTopicEnabled: "true"
     PULSAR_PREFIX_topicLevelPoliciesEnabled: "true"
+    # Enable the new delayed message
+    delayedDeliveryTrackerFactoryClassName: "org.apache.pulsar.broker.delayed.BucketDelayedDeliveryTrackerFactory"
+    # Enable transaction buffer segmented snapshot
+    transactionBufferSegmentedSnapshotEnabled: "true"
+    transactionBufferSnapshotSegmentSize: "262144"
+    # Enable the new Load Balancer
+    loadManagerClassName: "org.apache.pulsar.broker.loadbalance.extensions.ExtensibleLoadManagerImpl"
```

Comment:
To use `ExtensibleLoadManagerImpl`, we recommend upgrading the pulsar to 3.0.x before enabling the new load balancer.

https://pulsar.apache.org/docs/next/concepts-broker-load-balancing-migration/#migrate-from-modular-to-extensible-broker-load-balancer

cc @heesung-sn